### PR TITLE
Merge from Auth module  to main branch

### PR DESCRIPTION
**Pull request by NabhamDev**
Added ReadMe.md file comments and ingress controller

### CODE REVIEW

File: auth/src/index.ts [hunk 1]
```diff
@@ -4,6 +4,10 @@ import {json as bodyParser} from 'body-parser';
 const app=express();
 app.use(bodyParser());
 
+app.get('/api/users/currentuser',(req,res)=>{
```
Comment (Nacrosolutions) [inline auth/src/index.ts:9]: Specify which route handlers is required

**Review by Nacrosolutions — CHANGES_REQUESTED** [review]
Please add description in your ingress files and index.ts in auth module

File: auth/src/index.ts (hunk 1 above)
Comment (NabhamDev) [reply auth/src/index.ts:9]: ok sure !

**Comment by NabhamDev** [comment]
Added the comments as requested

**Review by Nacrosolutions — APPROVED** [review]

**Pull request merged**